lib/matplotlib/tests/test_ticker.py

```diff
@@ -467,6 +467,10 @@ class TestLogFormatter(object):
         (100000, 1000000.0, '1e5'),
     ]
 
+    call_data = [
+            1, 10, 100, 1000
+    ]
+
```

Comment:
I don't think this should be here. If you want to make a full-on fixture that is used in many tests, use pytest to mark it as such. Otherwise, just define this within the test that uses it.